### PR TITLE
 Updated documentation in the SignalFX exporter for translation rules

### PR DESCRIPTION
**Pull request by jvoravong**
Description:
Updated a couple translation descriptions to be more correct and detailed.

Link to tracking Issue:
I did not create an issue for this since these are documentation updates.

### CODE REVIEW

File: exporter/signalfxexporter/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Troubleshooting
```
Comment (dmitryax) [inline exporter/signalfxexporter/troubleshooting.md:1]: As discussed internally, this particular issue should be fixed without any customization required from user. Let's remove this section for now and keep comment fixes to the translation rules only

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@
 - `datadogexporter`: Add insecure_skip_verify flag to configuration (#7422)
 - `coralogixexporter`: Update readme (#7785)
 - `awscloudwatchlogsexporter`: Remove name from aws cloudwatch logs exporter (#7554)
+- `signalfxexporter`: Updated documentation in the SignalFX exporter for translation rules (#7875)
```
Comment (dmitryax) [inline CHANGELOG.md:15]: I think we don't need a change log line for comment fixes

**Review by dmitryax — COMMENTED** [review]
Also please update the PR description

File: CHANGELOG.md (hunk 2 above)
Comment (jvoravong) [reply CHANGELOG.md]: Removed

**Review by dmitryax — APPROVED** [review]

**Pull request merged**